src/marker.coffee

```diff
@@ -311,7 +310,6 @@ class Marker
   # * `screenPosition` The new {Point} to use
   # * `properties` (optional) {Object} properties to associate with the marker.
   setTailScreenPosition: (screenPosition, options) ->
-    screenPosition = @displayBuffer.clipScreenPosition(screenPosition, options)
```

Comment:
:sparkles: Clever deletion here: this runs already inside `displayBuffer.bufferPositionForScreenPosition`, so there's no need to run it a second time.